File: Python/fizzbuzz3.py
LIMIT = 100

a = ["FizzBuzz", "", "", "Fizz", "", "Buzz", "Fizz",
     "", "", "Fizz", "Buzz", "", "Fizz", "", ""]

for i in range(1, LIMIT + 1):
    print(a[i % 15] or i, end='')
    if i < LIMIT:
        print(', ', end='')

print()

File: Rust/fizzbuzz3.rs
const LIMIT: i32 = 100;

fn main() {
    println!("{}",
        (1..=LIMIT)
            .map(|i| {
                match i % 15 {
                    0 => String::from("FizzBuzz"),
                    3 | 6 | 9 | 12 => String::from("Fizz"),
                    5 | 10 => String::from("Buzz"),
                    _ => i.to_string(),
                }
            })
            .collect::<Vec<_>>()
            .join(", ")
    );
}
